### PR TITLE
added project section

### PR DESCRIPTION
**Pull request by shivams10**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,21 +14,27 @@
         <input type="checkbox" id="checkbox_toggle" />
         <label for="checkbox_toggle" class="hamburger">&#9776;</label>
         <div class="menu">
-          <li><a href="/">Home</a></li>
-          <li><a href="/">About</a></li>
-          <li><a href="/">Projects</a></li>
+          <li><a href="##home">Home</a></li>
```
Comment (siddhi-rajput-22) [inline index.html]: add single #

File: index.html [hunk 2]
```diff
@@ -79,5 +85,60 @@ <h3 class="headers">College</h3>
         </div>
       </div>
     </section>
+    <section class="project" id="project">
+      <div class="section-title"><span>Projects</span>
+        <div class="cards">
+          <div class="card">
+            <h2 class="card-title">Housing Society</h2>
+            <img
+              src="./assert//images//society.webp"
```
Comment (siddhi-rajput-22) [inline index.html]: change the src

File: index.html [hunk 3]
```diff
@@ -79,5 +85,60 @@ <h3 class="headers">College</h3>
...
+              <br />
+              <a href="https://github.com/shivams10/housing-society-frontend"
+                >Github Link >></a
+              >
+            </p>
+          </div>
+          <div class="card">
+            <h2 class="card-title">Cryptos</h2>
+            <img
+              src="./assert//images/cryptos.webp"
```
Comment (siddhi-rajput-22) [inline index.html:111]: same as line 94

File: index.html [hunk 4]
```diff
@@ -79,5 +85,60 @@ <h3 class="headers">College</h3>
...
+              using React js and api from coingecko.<br /><a
+                href="https://github.com/shivam910/cryptos"
+                >Github Link >></a
+              >
+            </p>
+          </div>
+          <div class="card">
+            <h2 class="card-title">Expense Tracker</h2>
+            <img
+              src="./assert//images/expense.jpeg"
```
Comment (siddhi-rajput-22) [inline index.html:126]: same as 94

File: index.html (hunk 1 above)
Comment (shivams10) [reply index.html]: i have added

File: index.html (hunk 2 above)
Comment (shivams10) [reply index.html]: okay doe

File: index.html (hunk 3 above)
Comment (shivams10) [reply index.html:111]: okay

File: index.html (hunk 4 above)
Comment (shivams10) [reply index.html:126]: okay

**Pull request closed without merging**